src/server/node.rs

```diff
@@ -105,6 +105,7 @@ where
         } else {
             store.set_address(cfg.advertise_addr.clone())
         }
+        store.set_version(env!("CARGO_PKG_VERSION").to_string());
```

Comment:
Can you explain how versions works? Why we should use `CARGO_PKG_VERSION`?

Reply:
@zhangjinpeng1987 PD needs it to handle the compatibility when rolling update.

Reply:
PD will maintain a cluster version. If rolling upgrade finished, the cluster version will increase. the Incompatible tikv with the cluster version will not not register successfully at startup.